www/class/centreonConfigCentreonBroker.php

```diff
@@ -745,6 +746,11 @@ public function insertConfig(array $values): bool
                 (int) $this->checkEventMaxQueueSizeValue($values['event_queue_max_size']),
                 \PDO::PARAM_INT
             );
+            $stmt->bindValue(
+                ':event_queues_total_size',
+                ! empty($values['event_queues_total_size']) ? (int) $values['event_queues_total_size'] : null,
+                \PDO::PARAM_INT
```

Comment:
Is there a difference between not value and zero ?
Because if `$values['event_queues_total_size'] === '0'` then `! empty($values['event_queues_total_size']) `will be false and the value zero will not be inserted into the database.

Reply:
No difference between no value and zero